server.cpp

```diff
@@ -38,6 +38,19 @@ std::string make_daytime_string()
     return buffer;
 }
 
+std::string int_to_hex(int dec)
+{
+    std::string digits[16] = {"0","1","2","3","4","5","6","7","8","9","A","B","C","D","E","F"};
```

Comment:
`const`

Reply:
Still needs `const`. Also, you can have an array of `char` rather than `std::string`.

Reply:
Also, since an array of chars is just a string you can have simply a string.